content/docs/ve/veNewCall.md

```diff
@@ -9,12 +9,14 @@ weight: 60
 ---
 ## Follow these instructions if you have updated your call sign with both the FCC and all VECs with which you are accredited.
 
-
-* Login to your account on https://ham.study
-* Change your callsign on https://ham.study/profile
+<br />
```

Comment:
I see you intentionally removed the blank line and add `br`. Why?

Reply:
It doesn't render an additional blank link in all cases without the <br /> tag. Markdown tries to be smart and combine multiple blank lines into a single carriage return. 

Officially markdown says "2 spaces at the end of a line and then enter" will insert an extra blank line; in testing, that didn't work. 

The goal is to create some extra white space between headers and list items so they aren't crowded.

Reply:
I saw that you removed a bunch of trailing blanks from one of the docs and was wondering why. Still wondering now...

Have you actually viewed this change locally in Hugo? It looks identical either way.

Reply:
Removing the tailing blank lines was just OCD at play; no actual reason.

Adding the ```<br />``` tag creates whitespace, where adding blank lines does not.  All examples below rendered with Hugo, locally.

***With ```<br />``` tag:***
![image](https://user-images.githubusercontent.com/68024347/102159897-7076b400-3e52-11eb-804d-09f4e78f7f5b.png)

***With 2 carriage returns with 2 spaces at the end (official MD whitespace), no ```<br />``` tags:***
![image](https://user-images.githubusercontent.com/68024347/102159997-9bf99e80-3e52-11eb-942d-21b643a54149.png)

***With characters at the beginning of the line, then 2 spaces, then a carriage return (just for reference):***
![image](https://user-images.githubusercontent.com/68024347/102160099-cba8a680-3e52-11eb-817a-f67ff883247e.png)


The desired result is the first example, so I put ```<br />``` in.